core/tasks.py

```diff
@@ -663,17 +700,19 @@ def _requeueStep(cls, qryDict: Dict[str, Any]) -> None:
             if req:
                 req.pendingTasks.append(task)  # < This property will be only exist on development server!
                 return
-        project = conf["viur.instance.project_id"]
-        location = queueRegion
-        parent = taskClient.queue_path(project, location, cls.queueName)
-        task = {
-            'app_engine_http_request': {  # Specify the type of request.
-                'http_method': 'POST',
-                'relative_uri': '/_tasks/queryIter'
-            }
-        }
-        task['app_engine_http_request']['body'] = json.dumps(preprocessJsonObject(qryDict)).encode("UTF-8")
-        taskClient.create_task(parent=parent, task=task)
+        taskClient.create_task(tasks_v2.CreateTaskRequest(
+            parent=taskClient.queue_path(conf["viur.instance.project_id"], queueRegion, cls.queueName),
+            task=tasks_v2.Task(
+                app_engine_http_request=tasks_v2.AppEngineHttpRequest(
+                    body=json.dumps(preprocessJsonObject(qryDict)).encode("UTF-8"),
+                    http_method=tasks_v2.HttpMethod.POST,
+                    relative_uri="/_tasks/queryIter",
+                    app_engine_routing=tasks_v2.AppEngineRouting(
+                        version=os.getenv("GAE_VERSION"),
```

Comment:
```suggestion
                        version=conf["viur.instance.app_version"],
```
You can use here a conf var.